extra/mariabackup/xtrabackup.cc

```diff
@@ -378,6 +378,10 @@ extern const char *innodb_checksum_algorithm_names[];
 extern TYPELIB innodb_checksum_algorithm_typelib;
 extern const char *innodb_flush_method_names[];
 extern TYPELIB innodb_flush_method_typelib;
+#ifdef __linux__
+extern const char *innodb_linux_aio_names[];
+extern TYPELIB innodb_linux_aio_typelib;
```

Comment:
Do we want backup to choose AIO implementation? Does really which AIO it uses, is it performance relevant? Can't we just use default one? Or even simulated one?

Reply:
It could be relevant for `mariadb-backup --prepare`. I’ll try to test a [MDEV-29911](https://jira.mariadb.org/browse/MDEV-29911) or [MDEV-34062](https://jira.mariadb.org/browse/MDEV-34062) like scenario to assess the impact.